rundoctests.py

```diff
@@ -9,11 +9,52 @@
 import os
 import sys
 import doctest
-import resource
+from doctest import DocTestParser, Example, SKIP
+from multiprocessing import Process
+if os.name != 'nt':
+    import resource
 
 flags = doctest.ELLIPSIS
+timeout = 600
+
+filenames = list(sys.argv[1:])
+if os.name == 'nt':
+    notinwindows = ['src/cysignals/pysignals.pyx',
+                    'src/cysignals/alarm.pyx',
+                    'src/cysignals/pselect.pyx']
+
+    for f in list(filenames):
```

Comment:
No need for explicit `list()`

Reply:
Maybe you meant the previous `list()`.  Here it's necessary since `filenames` is modified in the loop.